### DISCUSSION TITLE
why does parcel add hash in the name of compiled files?

### DISCUSSION THREAD

**Discussion by Darhagonable** [parcel-bundler/parcel#6330] [Q&A] [answered]
Is there any reason for why parcel adds the hash in the name of the files?
![image](https://user-images.githubusercontent.com/14127357/119229115-ae20ec80-bb16-11eb-9eac-c2ec0715ca85.png)

**Comment by DeMoorJasper** [comment, marked as answer]
This is so you can cache these files indefinitely on a cdn without any issues
